src/test/java/com/parasoft/findings/utils/results/testableinput/LocationUtilTest.java

```diff
@@ -0,0 +1,90 @@
+package com.parasoft.findings.utils.results.testableinput;
+
+import com.parasoft.findings.utils.results.violations.LocationsException;
+import com.parasoft.findings.utils.results.xml.IXmlTagsAndAttributes;
+import org.junit.jupiter.api.Test;
+import java.util.Properties;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class LocationUtilTest {
```

Comment:
Please change the method name to be short and clear:
`testCreateTestableInput_projIdAndResProjPathAttributesInProperties`-> `testCreateTestableInput_withProjIdAndResProjPath`
`testCreateTestableInput_scPathAttributeInProperties` -> `testCreateTestableInput_withScPath`
`testCreateTestableInput_SymIdAttributeInProperties` -> `testCreateTestableInput_withSymId`
`testCreateTestableInput_noSymIdAttributeInProperties` -> `testCreateTestableInput_noSymId`